activesupport/lib/active_support/encrypted_configuration.rb

```diff
@@ -22,6 +22,12 @@ def read
       ""
     end
 
+    def write(contents)
+      validate(contents)
+
+      super
+    end
```

Comment:
Should this be nodoc'd or documented? Same with `validate`.

Reply:
Since `EncryptedConfiguration` is a public, I thought it should be documented. I added doc.